compendium/DeclarativeServices/src/manager/states/CMDisabledState.hpp

```diff
@@ -86,7 +87,8 @@ namespace cppmicroservices
              * {\code CMDisabledState} created by the first thread. The second thread
              * receives the same future that was created by the first thread.
              */
-            std::shared_future<void> Disable(ComponentManagerImpl& cm) override;
+            std::shared_future<void> Disable(ComponentManagerImpl& cm,
+                                             std::shared_ptr<std::atomic<bool>> asyncStarted) override;
```

Comment:
doxygen comments need to be updated to describe the new behavior

Reply:
fixed